solr/core/src/test/org/apache/solr/cloud/ForceLeaderTest.java

```diff
@@ -305,8 +306,17 @@ private void doForceLeader(String collectionName, String shard)
       throws IOException, SolrServerException {
     CollectionAdminRequest.ForceLeader forceLeader =
         CollectionAdminRequest.forceLeaderElection(collectionName, shard);
+    boolean shardLeadersOnly = random().nextBoolean();
+    RandomizingCloudSolrClientBuilder builder =
```

Comment:
Again, lengthy types beg for use "var"

Reply:
I wish we had a commumnity standard for when to use a var...    or that `gw tidy` did it for me..  I went ahead and put one in.